src/shapes/view.turtle

```diff
@@ -120,21 +121,58 @@ view:Eq rdfs:label "=" .
                     sh:minCount 1 ;
                     sh:maxCount 1 ;
                     sh:order 2 ;
+                    sh:defaultValue view:In ;
                     sh:in
                         (
                             view:Gte
                             view:Lte
                             view:Lt
                             view:Gt
                             view:Eq
+                            view:In
```

Comment:
Currently rdf-cube-view-query has `view:Lang` as well https://github.com/zazuko/rdf-cube-view-query/blob/2321e607e88fa5822476867c2a91ec9cf4143aa5/lib/FilterBuilder.js#L37 should we add it?

Reply:
same as `view:Ne`

Reply:
I would not do that in the PR. The comparison operators, `view:In` and `view:Lang` all have different semantics and would need to be handled using different UI controls